Makefile

```diff
@@ -184,14 +186,20 @@ ko-build-push: test ## Build and push container image with ko
 # this is used for local testing 
 .PHONY: kind-load-image
 kind-load-image: 
-	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${IMG}
+	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${CORE_IMAGE_REPO}:${BUILD_TIME_TAG}
 
 .PHONY: chart
 chart: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${IMG} | sed 's/\(:.*\)v/\1/')
 	$(KUSTOMIZE) build config/default > chart/templates/controller-manager.yaml
 	scripts/add-helm-code.sh add
 
+.PHONY: local-chart
+local-chart: manifests kustomize
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${CORE_IMAGE_REPO}:${BUILD_TIME_TAG} | sed 's/\(:.*\)v/\1/')
+	$(KUSTOMIZE) build config/default > chart/templates/controller-manager.yaml
+	scripts/add-helm-code.sh add
```

Comment:
Rather than double some Makefile contents, wouldn't it be simpler to default `IMG` to using the local tag?

The release process seems to be headed toward not having contributors use `make` to create and publish the published images.

(I would find it comforting if the Make variable names IMG and IMAGE_TAG were changed to make it clear that these are just for local usage.)

Reply:
The release process is not going to use make?  If that's the case, we can retire IMG all together.

Reply:
see example for oct-status-addon - the idea is to have the same approach here: 
https://github.com/kubestellar/ocm-status-addon/blob/ab39859c91b65e6f11ce84a5f3ee557c156890b6/Makefile#L185-L199

and 

https://github.com/kubestellar/ocm-status-addon/blob/ab39859c91b65e6f11ce84a5f3ee557c156890b6/Makefile#L16-L17

Reply:
and still use make chart:

https://github.com/kubestellar/ocm-status-addon/blob/ab39859c91b65e6f11ce84a5f3ee557c156890b6/.github/workflows/goreleaser.yml#L60-L64

Reply:
Thanks @pdettori that helped cleared things a great deal.  I updated the Makefile accordingly.